### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Access Key Purpose_Screen Reader: Narrator is not reading the controls present under "Expires in" label.

### ISSUE THREAD

**Issue by Accessibility56** [Microsoft/IIS.WebManager#75]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #.NETWebsites; #ScreenReader; #MAS1.3.2; #Narrator; #NVDA; #C+E-H; #Chrome; #Edge

**Environment Details:**
URL: https://localhost:55539/security/tokens
Browser Details: Edge(42.17134.1.0), Chrome(68.0.3440.106)
Screen Reader: Narrator, NVDA
**MAS ID Violated:** MAS 1.3.2

**Repro Steps:**
1.Open "Microsoft IIS" application using the URL: https://manage.iis.net/
2.Download and install "Microsoft IIS Administration" application.
3.Tab till "Skip This" button and Press Enter.
4.Tab till "Get Access Token" link and Press Enter.
5.Tab till "Create Access Token" button and press enter.
6.Start Narrator.
7.Verify narrator is reading all controls present inside "Access Key Purpose" pop up.

**Actual Result:**
Narrator focus is not going to the controls present under " Expires in" label.
**NOTE:** Issue is observed in "Chrome" as well.

**Expected Result:**
Narrator focus should go to each and every interactive element on page while tabbing.

**User Impact:**
Screen Reader users can not get complete information regarding application if narrator focus is not going to the all controls on page.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b}

**Attachment:**
[Bug_75_Access Key Purpose_Screen Reader Issue.pptx](https://github.com/Microsoft/IIS.WebManager/files/2317988/Bug_75_Access.Key.Purpose_Screen.Reader.Issue.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by fsteffi** [comment]
#A11ySev1;

**Comment by fsteffi** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will  be removed.
